src/main/java/org/broadinstitute/hellbender/tools/PrintReads.java

```diff
@@ -13,7 +13,33 @@
 import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
 
 /**
- * Print reads from a SAM/BAM/CRAM file.
+ * Copy reads in a SAM/BAM/CRAM file verbatim to a SAM file
```

Comment:
This makes it sound like it only outputs sam files.  Shouldn't it say to a SAM/BAM/CRAM file?